book.rb

```diff
@@ -0,0 +1,8 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
+end
```

Comment:
- [ ] Great job so far. However, there're few things to add here. There should be many-to-many relationship between `Person` and `Book` using the intermediate class `Rental`.
- A book has many rental
Something similar to this :point_down: 

```
  class Book
  
  # *Getters and Setters for @rentals should be present
  attr_accessor :title, :author, :rentals check this: (https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor) for more info.
  .
  .

  def initialize(title, author)
    .
    .
    # *@rentals should be initialized. You can check this link: https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars
    @rentals = []
  end
  
  # *A method for adding rentals should be present. You can check this link too: https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship
  def add_rental(person, date)
    # *the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 
    Rental.new(date, self, person)
  end
end

```